app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md

```diff
@@ -160,6 +160,10 @@ There are usually more full-time courses available than part-time.
 
 Postgraduate teacher training usually takes 9 months full-time, or 18 to 24 months part-time.
 
+## Adjustments and accessibility
+
+Course providers can make adjustments if you need supoprt because you're neurodiverse, have a long-term physical or mental health condition, or have any other accessibility needs. Find out more about [adjustments to help you train](/train-to-be-a-teacher/accessibility-adjustments).
```

Comment:
```suggestion
Course providers can make adjustments if you need support because you're neurodiverse, have a long-term physical or mental health condition, or have any other accessibility needs. Find out more about [adjustments to help you train](/train-to-be-a-teacher/accessibility-adjustments).
```